app/models/concerns/notify_concern.rb

```diff
@@ -0,0 +1,70 @@
+module NotifyConcern
```

Comment:
Wrap this inside a function like <code>acts_as_notifiable</code>, so others can include you by:

```ruby
class Achievement
  acts_as_notifiable  # instead of include
end
```